### ISSUE TITLE
HELP: New maintainer needed

### ISSUE THREAD

**Issue by jakeasmith** [jakeasmith/http_build_url#27]
I haven't written PHP in years and it seems there are people still using this. If someone from the PHP community is interested in taking over this I will happily hand it over. This package does have a surprising amount of monthly downloads, so while I don't have a particular process in mind, it would be good to see a few people vouch for the new maintainer.

**Comment by ozh** [comment]
Count me in. My [main project](https://github.com/YOURLS/YOURLS) [uses](https://github.com/YOURLS/YOURLS/blob/master/composer.json#L27) yours, I've [contributed](https://github.com/jakeasmith/http_build_url/graphs/contributors) a bit to yours, happy to continue :)

**Comment by ozh** [comment]
Ping

**Comment by LukeTowers** [comment]
I've just put in a ton of work on our (@WinterCMS's) wrapper for `http_build_url()` (inspired heavily by yours and @ivantcholakov's; see https://github.com/wintercms/storm/commit/05e8dc11700303a0ef5f5f1210b1682574b38616 for the latest commit on that front. If you'd like to hand maintaining the package over to a team of highly experienced PHP developers we could easily split out our implementation into a separate package and provide that as the recommended replacement for this package moving forwards.

Let me know if you're interested in doing that @jakeasmith!

**Comment by jawira** [comment]
@jakeasmith Yes please 🙋‍♂️ I want to be the new maintainer. I use `http_build_url()`, I also have experience maintaining PHP packages.

**Comment by ozh** [comment]
Ping, ping, ping. See https://github.com/ozh/http_build_url/ I'm maintaining it because using it for a 12K star project

**Comment by jawira** [comment]
If you are in a project with >=PHP 8.5, you can replace the abandoned `http_build_url` library with `Uri\Rfc3986\Uri`.

```php
use Uri\Rfc3986\Uri;

$uri = (new Uri())
    ->withScheme('https')
    ->withHost('example.com')
    ->withPath('/api/users')
    ->withQuery('page=2');

echo $uri->toString();
```

Source: https://sensiolabs.com/blog/2025/php-85-new-uri-extension

**Comment by jakeasmith** [comment]
Sorry for going quiet on this. I asked for help and never followed up. Life got in the way. Thank you @ozh, @LukeTowers, and @jawira for offering, and @ozh for keeping a fork going for YOURLS.

I've decided to deprecate the package instead of handing it over. As @jawira pointed out, PHP 8.5 ships Uri\Rfc3986\Uri, which does this job natively, and I'd rather get people off this and onto that.

1.0.2 marks the package abandoned, and the README now shows how to replace http_build_url() with the new URI API. Existing versions stay installable. I'm closing the open issues and PRs. Thank you again for your support <3

**Comment by ozh** [comment]
For the record : I'll keep maintaing [ozh/http_build_url](https://github.com/ozh/http_build_url/) till my main project is 8.5+ (which would happen in several years, probably)
